entrypoint.sh

```diff
@@ -19,25 +19,24 @@ if [ ! -z "$OKTETO_CA_CERT" ]; then
 fi
 
 if [ ! -z "$log_level" ]; then
-  log_level="--log-level ${log_level}"
+  params="${params} --log-level ${log_level}"
 fi
 
 # https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
 # https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
 if [ "${RUNNER_DEBUG}" = "1" ]; then
-  log_level="--log-level debug"
+  params="${params} --log-level debug"
```

Comment:
This is wrong. If I specify the `log-level` flag, and the debug option in the github Action UI, it will send 2 `log-level` flags, and I don't know if it would fail or not, or which log level would use. That is why `log-level` is not part of `params` in the rest of actions.

I'd follow the same pattern we have in the rest of actions